pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```

Comment:
Typically, this log only output once when PodCIDR is set though. It will not happened repeatedly. See line #120 of this file. And it makes folks life much easier to debug CNI stuff, given it is already tough.